backend/cms/models/poi.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+
+class POI(models.Model):
+    address = models.CharField(max_length=250)
```

Comment:
Whoops. Here's an association to a Site missing.

Reply:
As far as I'm not missing something the assiciations to a Site are missing in every model so far (Events, POIs, Pages).

Reply:
Yeah, I've added the Site-associations in this PR, but forget about the POI model...